### DISCUSSION TITLE
Why Mastodon chose the name "Status" instead of "Post" for their model?

### DISCUSSION THREAD

**Discussion by sergioro9** [mastodon/mastodon#27495] [Q&A] [answered]
I think "Post" would be a shorter and more descriptive name for Mastodon's "Status" model.

My guess is that Mastodon used "Status" because Mastodon was based on OStatus protocol which refers to posts as statuses.

Would there be any drawbacks with renaming such model to Post?

**Comment by Gargron** [comment, marked as answer]
It's not just OStatus. Microblogging emerged from status updates on IM.

There is no benefit to renaming the model. Only developers see the model name. Users know them as "posts".

**Comment by nightpool** [comment]
it would probably take nearly a hundred programmer-hours to rename the model and all of the usages of the term `status` in the codebase, for no ux or dx benefit. Indeed, the word `post` is suboptimal for programming because it can be both a verb _and_ a noun, which can lead to ambiguity in many situations (such as method names). status is only ever a noun, which lets us use names like PostStatusService without any awkward repetition or ambiguity.
